contracts/main/proxy-wallet/ProxyWallet.sol

```diff
@@ -3,33 +3,23 @@ pragma solidity 0.8.17;
 
 import "./FathomAuth.sol";
 import "./FathomNote.sol";
-import "./ProxyWalletCache.sol";
```

Comment:
hm... why goodbye to cache? I need to look at the code to understand what it was doing